packages/edge-gateway/src/durable-objects/summary-metrics.js

```diff
@@ -4,19 +4,23 @@ import {
 } from '../utils/histogram.js'
 
 /**
+ * @typedef {'CID'|'CID+PATH'} QueryType
```

Comment:
I'd maybe not use "CID" here since IPNS keys are CIDs too. Maybe something like:

* IPFS direct
* IPFS pathed
* IPNS direct
* IPNS pathed